File: servers/jax-rs/src/main/java/com/wordnik/client/model/StatsHistory.java
package com.wordnik.client.model;

import java.util.Date;
public class StatsHistory {
  private Date date = null;
  private String rootSymbol = null;
  private String currency = null;
  private Double volume = null;
  private Double turnover = null;
  public Date getDate() {
    return date;
  }
  public void setDate(Date date) {
    this.date = date;
  }

  public String getRootSymbol() {
    return rootSymbol;
  }
  public void setRootSymbol(String rootSymbol) {
    this.rootSymbol = rootSymbol;
  }

  public String getCurrency() {
    return currency;
  }
  public void setCurrency(String currency) {
    this.currency = currency;
  }

  public Double getVolume() {
    return volume;
  }
  public void setVolume(Double volume) {
    this.volume = volume;
  }

  public Double getTurnover() {
    return turnover;
  }
  public void setTurnover(Double turnover) {
    this.turnover = turnover;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class StatsHistory {\n");
    sb.append("  date: ").append(date).append("\n");
    sb.append("  rootSymbol: ").append(rootSymbol).append("\n");
    sb.append("  currency: ").append(currency).append("\n");
    sb.append("  volume: ").append(volume).append("\n");
    sb.append("  turnover: ").append(turnover).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: clients/csharp/src/Swagger/Client/Model/StatsHistory.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Swagger.Client.Model {
  public class StatsHistory {
    public DateTime? date { get; set; }

    public string rootSymbol { get; set; }

    public string currency { get; set; }

    public double? volume { get; set; }

    public double? turnover { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class StatsHistory {\n");
      sb.Append("  date: ").Append(date).Append("\n");
      sb.Append("  rootSymbol: ").Append(rootSymbol).Append("\n");
      sb.Append("  currency: ").Append(currency).Append("\n");
      sb.Append("  volume: ").Append(volume).Append("\n");
      sb.Append("  turnover: ").Append(turnover).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
